packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2062,6 +2113,21 @@ class AnnotatedRegionLayer<T> extends ContainerLayer {
   /// Ignored if [size] is not set.
   final Offset offset;
 
+  /// Whether this layer should prevent layers visually behind it from receiving
+  /// the hit testing.
```

Comment:
`receiving the hit testing` -> `being visible to hit tests` or some such

Reply:
or "being hit tested"

Reply:
I changed it to `being to the hit during a hit test.`